packages/account-export/src/lib/helpers/importAccount.ts

```diff
@@ -0,0 +1,62 @@
+import CryptoJS from "crypto-js";
+import generator from "generate-password";
+
+import type { AccountImportData } from "@near-wallet-selector/core";
+
+export const encodeAccountData = (
+  accountsData: Array<AccountImportData>
+): string => encodeURIComponent(JSON.stringify(accountsData));
+
+export const decodeAccountData = (
+  encodedAccountsData: string
+): Array<AccountImportData> =>
+  JSON.parse(decodeURIComponent(encodedAccountsData));
+
+interface EncryptAccountDataProps {
+  accountData: Array<AccountImportData>;
+  secretKey: string;
+}
+
+export const encryptAccountData = ({
+  accountData,
+  secretKey,
+}: EncryptAccountDataProps): CryptoJS.lib.CipherParams => {
```

Comment:
Ideally this would just return the encrypted string and the decryption method would take an encrypted string as well. Since we're exposing parts of this in the API we should keep the interface as simple as possible.

Is there a compelling reason to have it return a `CryptoJS` object?